rcljava/src/main/java/org/ros2/rcljava/node/NodeImpl.java

```diff
@@ -480,6 +480,8 @@ public final String getName() {
           new rcl_interfaces.msg.ListParametersResult();
 
       String separator = ".";
+      List<String> resultNames = new ArrayList<String>();
+      List<String> resultPrefixes = new ArrayList<String>();
```

Comment:
are the changes to this file related to the PR?
I don't see a connection

Reply:
Yes, they are related. The members of `result` have changed to array types, therefore we can't do something like:

```java
result.getNames().add(...);
```

The following also won't work, since it is returning a copy:

```java
result.getNamesAsList().add(...);
```